LoRaEngine/LoraKeysManagerFacade/IoTCentralImp/Definitions/Device.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoraKeysManagerFacade.IoTCentralImp.Definitions
```

Comment:
On top of the previous comments about IoT Central vs DPS, I would avoid abbreviations. So IotCentral should be enough or DeviceProvisioningService. In this case Imp (or Implementation) doesn't really bring any value in the namespace. And I would challenge as well Definitions. What real value does it bring?